.ci/bump-cs-version-in-sevntu-checks.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -e
+
+if [[ -z $1 ]]; then
+  echo "ERROR: version is not set"
+  echo "Usage: bump-cs-version-in-gradle-project.sh <version>"
+  exit 1
+fi
+
+VERSION=$1
+cd sevntu-checks
+mvn versions:set-property -Dproperty=checkstyle.version -DnewVersion="$VERSION"
+echo "Version updated to $VERSION at sevntu-checks"
```

Comment:
sevntu-checks/pom.xml

Reply:
please also remove bak up file https://github.com/sevntu-checkstyle/checkstyle-samples/blob/4b0759612d8230cacaaf0f7bd88b61747d21080f/.ci/bump-cs-version-in-maven-project.sh#L14

Reply:
not done

Reply:
Changes Done